tools/debian/rules

```diff
@@ -8,7 +8,13 @@ PRE4AA = $(filter $(shell . /etc/os-release; echo $${VERSION_ID:-unstable}),22.0
 # pcp isn't in Debian testing right now
 ifeq ($(shell . /etc/os-release; echo $${VERSION_CODENAME}),trixie)
     export DH_OPTIONS = -Ncockpit-pcp
-    CONFIG_OPTIONS = --disable-pcp
+    CONFIG_OPTIONS := --disable-pcp
+endif
+
+# keep the host switcher on in currently supported stable releases
+STABLE = $(filter $(shell . /etc/os-release; echo $${VERSION_CODENAME}),bookworm jammy noble)
+ifeq ($(STABLE),)
+    CONFIG_OPTIONS := $(CONFIG_OPTIONS) --enable-multihost
```

Comment:
That condition looks the wrong way around to me now -- we want `--enable-multihost` precisely for bookworm/jammy/noble, i.e. if teh filter is *not* equal to an empty string, no? But my brain isn't 100% so maybe I'm just  confused.

Reply:
`ifeq ($(STABLE),)` is true when $(STABLE) is the empty string, and it is the empty string when codename is not in the given list of stable versions. Yeah, confusing.

Reply:
Ouch, my confusion.